### PR TITLE
DOCS: Fix internal links to other functions in isomorphvf2

### PR DESCRIPTION
**Pull request by MridulS**

### CODE REVIEW

File: networkx/algorithms/isomorphism/isomorphvf2.py [hunk 1]
```diff
@@ -21,8 +21,8 @@
 as the logical AND of the two functions.
 
 To include a semantic check, the (Di)GraphMatcher class should be
-subclassed, and the semantic_feasibility() function should be
-redefined.  By default, the semantic feasibility function always
+subclassed, and the :func:`~networkx.isomorphism.isomorphvf2.GraphMatcher.semantic_feasibility`
```
Comment (rossbar) [inline networkx/algorithms/isomorphism/isomorphvf2.py:24]: These are still not rendering as links in the docs, maybe they need to be `:meth:` roles?

Comment (MridulS) [reply networkx/algorithms/isomorphism/isomorphvf2.py]: Urgghh, it still doesn't do the proper linking. Any other ideas ?

**Comment by MridulS** [comment]
@rossbar the new changes should fix the links.

**Review by rossbar — APPROVED** [review]
Thanks @MridulS !

**Pull request merged**